### DISCUSSION TITLE
Import and export is brocken

### DISCUSSION THREAD

**Discussion by wotomg** [docmost/docmost#368] [Help / Questions]
Importing and exporting only works with the page header. The page body is not affected.

**Comment by Philipinho** [comment]
What version are you running?

Check the`/collab` path in the “WS” websocket tab in your browser console. Confirm if it’s transmits data when you type on the editor.

**Comment by wotomg** [comment]
It was 0.2.10. Now I have tested in 0.3.1, same issue.
